index.html

```diff
@@ -1,15 +1,31 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
```

Comment:
No quites el idioma a la pagina, es uno de los puntos importantes para hacerla accesible
```suggestion
<html lang="es">
```